pkgs/misc/emulators/wine/winetricks.nix

```diff
@@ -1,25 +1,31 @@
-{ stdenv, fetchsvn, wine, perl, which, coreutils, zenity, curl
+{ stdenv, fetchgit, wine, perl, which, coreutils, zenity, curl
 , cabextract, unzip, p7zip, gnused, gnugrep, bash } :
 
 stdenv.mkDerivation rec {
-  rev = "1199";
-  name = "winetricks-${rev}";
+  name = "winetricks";
```

Comment:
Thanks for reminding! I've used to left vcs-sourced packages without a version earlier but this is against our packaging style.